dev-db/tokyocabinet/tokyocabinet-1.4.48-r1.ebuild

```diff
@@ -28,17 +29,18 @@ src_prepare() {
 	sed -i -e 's/-fomit-frame-pointer//' configure.in || die
 	# flag only works on x86 derivatives, remove everywhere else
 	if ! use x86 && ! use amd64; then sed -i -e 's/ -minline-all-stringops//' configure.in; fi
+	sed -e 's/libtokyocabinet.a/libtokyocabinet.so/g' -i configure.in || die
 	eautoreconf
 }
 
 src_configure() {
 	# we use the "fastest" target without the -O3
 	econf \
-	$(use_enable debug) \
 	$(use_enable bzip2 bzip) \
-	$(use_enable zlib) \
+	$(use_enable debug) \
 	$(use_enable threads pthread) \
-	--enable-off64 --enable-fastest
+	$(use_enable zlib) \
+	--enable-off64 --enable-fastest	--disable-static
 }
```

Comment:
revbump instead of inplace
1. move the patch into `PATCHES`, add `a/` and `b/` and call `default` first thing in `src_prepare`
2. `src_install` needs a lot of changing:
```
src_install() {
	default

	use doc && dodoc -r doc/.
	if use examples; then
		docinto examples
		dodoc -r example/.
		docompress -x /usr/share/doc/${PF}/examples
	fi
}
```
3. `src_test` should come before `src_install` and remove the `|| die`

Reply:
I read that the documentation should be installed anyway if doesn't need to be built

Reply:
depends on how large `doc/` is. 500KiB? install unconditionally. 500MiB? definitely not. Use good judgement

Reply:
It's 1,5 MiB in this case

Reply:
I'll leave it conditional because it also install Japanese documentation